libraries/botbuilder-ai-orchestrator/src/orchestratorAdaptiveRecognizer.ts

```diff
@@ -402,18 +392,15 @@ export class OrchestratorAdaptiveRecognizer
         // Add full entity recognition result as a 'result' property
         recognizerResult[this.entityProperty] = results;
         if (results.length) {
-            if (recognizerResult.entities === null) {
-                recognizerResult.entities = {};
-            }
+            recognizerResult.entities ??= {};
 
             results.forEach((result: Result) => {
                 const entityType = result.label.name;
 
                 // add value
                 let values: any[] = recognizerResult.entities[entityType];
```

Comment:
Can you make this `unknown[]` instead of `any[]`? Using `any` is generally bad practice.

Reply:
fixed the block with 

const values = recognizerResult.entities[entityType] ?? [];